.github/workflows/unit-tests.yml

```diff
@@ -11,4 +11,4 @@ jobs:
         uses: impress-org/givewp-github-actions/.github/workflows/addon-tests.yml@master
         with:
             addon_slug: givewp-next-gen
-            givewp_branch: develop
+            givewp_branch: feautre/show-in-admin
```

Comment:
@jonwaldstein Are we going to set this back after this is merged?

Reply:
yesszir